sdk/identity/identity/src/credentials/clientSecretCredential.ts

```diff
@@ -17,23 +16,22 @@ export class ClientSecretCredential extends AzureCredential {
     clientSecret: string,
     options?: IdentityClientOptions
   ) {
-    super(options);
-
+    this.identityClient = new IdentityClient(options);
     this._tenantId = tenantId;
     this._clientId = clientId;
     this._clientSecret = clientSecret;
   }
 
-  protected getAccessToken(
+  public getToken(
```

Comment:
So.. ClientSecretCredential doesnt use cache token anymore?

Reply:
`BearerTokenAuthenticationPolicy` now handles token caching, we've gotten rid of the `AzureCredential` class that used to handle that